nxc/protocols/winrm.py

```diff
@@ -170,34 +175,56 @@ def laps_search(self, username, password, ntlm_hash, domain):
 
     def print_host_info(self):
         if self.args.domain:
-            self.logger.extra["protocol"] = "HTTP"
+            self.logger.extra["protocol"] = "HTTPS" if self.ssl else "HTTP"
             self.logger.display(self.endpoint)
         else:
             self.logger.extra["protocol"] = "SMB"
+            self.logger.extra["port"] = "445"
             self.logger.display(f"{self.server_os} (name:{self.hostname}) (domain:{self.domain})")
-            self.logger.extra["protocol"] = "HTTP"
+            self.logger.extra["protocol"] = "HTTPS" if self.ssl else "HTTP"
+            self.logger.extra["port"] = self.port
```

Comment:
It's defined as 445 above, but then redefined here, can you put it into one assignment to default to 445 if self.port isnt defined

Reply:
Sure

Reply:
Oh, that mind be wrong, because `self.port` always in `connection.py` and it always existed

Reply:
Maybe like this?

Reply:
https://github.com/Pennyw0rth/NetExec/pull/72/commits/e85fe3c4f2d08f2451827b24d08b39f8dacbd3e2

Reply:
Solved!